tests/gql_behave/tests/memgraph_V1/features/property_caching.feature

```diff
@@ -0,0 +1,167 @@
+Feature: MapLiteral property caching
```

Comment:
I think the naming here is wrong. from a gqlbehave standpoint, we do not need to know the implementation details. Just verify the correctness and name it from the SetProperties standpoint

Reply:
Done